scripts/3_visualize/visualizeLandUse.R

```diff
@@ -31,12 +32,7 @@ gsplotLandUseConc <- function(fname.data){
     rect(geom.df$x.left, geom.df$y.bottom, 
          geom.df$x.right, geom.df$y.top,
          lwd=0.5, col = geom.df$rect.col, 
-         legend.name=levels(geom.df$type)) %>% 
-    axis(side = 1, at = geom.df$x.middle, 
-         labels = geom.df$site.label, 
-         tick = FALSE, las = 2, cex.axis = 0.1) %>% 
-    axis(side = 2, at = seq(0, 10, by=5))
```

Comment:
@jread-usgs I know they are sharing an x-axis, but should we be keeping the yaxis formatting for this plot?